src/sentry/scim/endpoints/teams.py

```diff
@@ -197,10 +198,11 @@ def post(self, request: Request, organization) -> Response:
         return super().post(request, organization)
 
 
+@extend_schema(tags=["SCIM"])
 @region_silo_endpoint
 class OrganizationSCIMTeamDetails(SCIMEndpoint, TeamDetailsEndpoint):
     permission_classes = (OrganizationSCIMTeamPermission,)
-    public = {"GET", "PATCH"}
+    public = {"GET", "PATCH", "DELETE"}
```

Comment:
@maxiuyuan Do we want to make DELETE public?

Reply:
i was wondering this too because the docs for DELETE don't generate unless DELETE is public

Reply:
Yeah, we only generate docs for Public methods, but it also means we're committing to support it.

Reply:
DELETE has public docs but those were generated from the static docs https://docs.sentry.io/api/scim/delete-an-individual-team/